File: javascript-exercicios/ex052.js
const prompt = require("prompt-sync")()
let num = Number(prompt("digite um número: "))
let tot = 0
for (let c=1; c<num; c++)
{
    if (num % c == 0){
        tot += 1
    }
}
console.log(`o número ${num} foi dividido ${tot} vezes`)
if(tot==2){
    console.log("por isso que é primo")
}else{
    console.log("por isso que não é primo")
}

File: curso-python/ex052.py
num = int(input("digite um número: "))
tot = 0
for c in range(1, num+1):
    if num % c == 0:
        print('\033[33m', end='')
        tot += 1
    else:
        print('\033[31m', end='')
    print("{}".format(c),end=' ')

print(f"\n\033[mo número {num} foi dividido {tot} vezes")
if tot == 2:
    print("por isso que é primo")
else:
    print("por isso que não é primo")